### PR TITLE
 app-emulation/lxd: bump to 2.21

### PR DESCRIPTION
**Pull request by hsoft**
Closes: https://bugs.gentoo.org/641884
Package-Manager: Portage-2.3.13, Repoman-2.3.3

### CODE REVIEW

**Comment by hsoft** [comment]
ping @tamiko

**Comment by hsoft** [comment]
This bump is a bit special because I had to significantly improve my dependencies management scripts due to significant upstream dependencies changes: https://github.com/hsoft/gentoo-ego-vendor-update/commit/fba61f5ed0528a8a2cb54295b1dc333482538162

File: app-emulation/lxd/lxd-2.21.ebuild [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+	use nls && emake build-mo
+}
+
+src_test() {
+	if use daemon; then
+		export GOPATH="${S}"
+		cd "${S}/src/${EGO_PN}" || die "Failed to change to deep src dir"
+
+		emake check
+	fi
```
Comment (titanofold) [inline app-emulation/lxd/lxd-2.21.ebuild]: else
         einfo "No tests to run for client-only builds"

Comment (hsoft) [reply app-emulation/lxd/lxd-2.21.ebuild]: Thanks for the tip @titanofold . done. I've also verified that it works as expected.

**Pull request closed without merging**